pkg/crmanager/backend.go

```diff
@@ -386,6 +386,13 @@ func createServiceDecl(cfg *ResourceConfig, sharedApp as3Application) {
 	svc.TranslateServerAddress = true
 	svc.TranslateServerPort = true
 
+	if cfg.Virtual.SNAT == "auto" || cfg.Virtual.SNAT =="none" {
```

Comment:
This code block is not formatted properly.

Reply:
Addressed formatting issue